src/ConvertInputFormat/CMakeLists.txt

```diff
@@ -26,3 +26,67 @@ set_target_properties(ConvertInputFormat PROPERTIES VERSION ${ENERGYPLUS_VERSION
+  set_tests_properties(ConvertInputFormat.Setup.CopyNoDot PROPERTIES DEPENDS ConvertInputFormat.Setup.RemakeDir FIXTURES_SETUP ConvertInputFormat.SetupFixture)
+
+
+  add_test(NAME ConvertInputFormat.RegularFile_AbsolutePath
+    COMMAND ConvertInputFormat --output "${TEST_DIR}/RegularFile_AbsolutePath" "${TEST_DIR}/1ZoneUncontrolled.idf"
+  )
+  add_test(NAME ConvertInputFormat.RegularFile_AbsolutePath.TestOK
+    COMMAND ${CMAKE_COMMAND} -E cat "${TEST_DIR}/RegularFile_AbsolutePath/1ZoneUncontrolled.epJSON" # cat will fail if it doesn't exist
+  )
+  set_tests_properties(ConvertInputFormat.RegularFile_AbsolutePath.TestOK PROPERTIES FIXTURES_CLEANUP RegularFile_AbsolutePathFixture)
+  set_tests_properties(ConvertInputFormat.RegularFile_AbsolutePath PROPERTIES FIXTURES_REQUIRED "ConvertInputFormat.SetupFixture; RegularFile_AbsolutePathFixture")
+
+
+  add_test(NAME ConvertInputFormat.RegularFile_RelativePath
+    COMMAND ConvertInputFormat --output RegularFile_RelativePath "1ZoneUncontrolled.idf"
+    WORKING_DIRECTORY ${TEST_DIR}
+  )
+  add_test(NAME ConvertInputFormat.RegularFile_RelativePath.TestOK
+    COMMAND ${CMAKE_COMMAND} -E cat "${TEST_DIR}/RegularFile_RelativePath/1ZoneUncontrolled.epJSON" # cat will fail if it doesn't exist
+  )
+  set_tests_properties(ConvertInputFormat.RegularFile_RelativePath.TestOK PROPERTIES FIXTURES_CLEANUP RegularFile_RelativePathFixture)
+  set_tests_properties(ConvertInputFormat.RegularFile_RelativePath PROPERTIES FIXTURES_REQUIRED "ConvertInputFormat.SetupFixture; RegularFile_RelativePathFixture")
+
+
+
+  add_test(NAME ConvertInputFormat.ExtraDotFile_AbsolutePath
+    COMMAND ConvertInputFormat --output "${TEST_DIR}/ExtraDotFile_AbsolutePath" "${TEST_DIR}/1.ZoneUncontrolled.idf"
+  )
+  add_test(NAME ConvertInputFormat.ExtraDotFile_AbsolutePath.TestOK
+    COMMAND ${CMAKE_COMMAND} -E cat "${TEST_DIR}/ExtraDotFile_AbsolutePath/1.ZoneUncontrolled.epJSON" # cat will fail if it doesn't exist
+  )
+set_tests_properties(ConvertInputFormat.ExtraDotFile_AbsolutePath.TestOK PROPERTIES FIXTURES_CLEANUP ExtraDotFile_AbsolutePathFixture)
+  set_tests_properties(ConvertInputFormat.ExtraDotFile_AbsolutePath PROPERTIES FIXTURES_REQUIRED "ConvertInputFormat.SetupFixture; ExtraDotFile_AbsolutePathFixture")
+
+
+  add_test(NAME ConvertInputFormat.ExtraDotFile_RelativePath
+    COMMAND ConvertInputFormat --output ExtraDotFile_RelativePath "1.ZoneUncontrolled.idf"
+    WORKING_DIRECTORY ${TEST_DIR}
+  )
+  add_test(NAME ConvertInputFormat.ExtraDotFile_RelativePath.TestOK
+    COMMAND ${CMAKE_COMMAND} -E cat "${TEST_DIR}/ExtraDotFile_RelativePath/1.ZoneUncontrolled.epJSON" # cat will fail if it doesn't exist
+  )
+  set_tests_properties(ConvertInputFormat.ExtraDotFile_RelativePath.TestOK PROPERTIES FIXTURES_CLEANUP ExtraDotFile_RelativePathFixture)
+  set_tests_properties(ConvertInputFormat.ExtraDotFile_RelativePath PROPERTIES FIXTURES_REQUIRED "ConvertInputFormat.SetupFixture; ExtraDotFile_RelativePathFixture")
+
+endif()
```

Comment:
Added some ctest for ConvertInputFormat... This makes use of CTest **Fixtures**. This isn't terribly fun to read but I'm just basically doing

* Setup fixtures: remove any existing dir, remake dirs. Then copy 1ZoneUncontrolled to the test dir
* Run ConvertInput format on that file
* Cleanup fixture: asserts that the epJSON created has the correct name by using `cmake -E cat` on it (that's a cross platform workaround...).